azure-mqtt/cpp/main.cpp

```diff
@@ -51,22 +54,8 @@
 #include <grove.hpp>
 #endif
 
-/**
- * This project template allow you to test the Microsoft* Azure* IoT Hub.
- * It will read the temperature from the Grove Temperature Sensor and send
- * it to the Azure* IoT Hub.
- *
- * First steps:
- * - Create an IoT HuB following the instruction in
- * https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal
- * - Create a connection string for a device following the instruction in
- * https://aka.ms/manageiothub
- * - Copy the connection string inside the "credentials.h" header
- * - Run the example and check the results inside your Azure* IoT Hub
- *
- * * Other names and brands may be claimed as the property of others.
```

Comment:
As per Tom's recent usability study, users like having setup instructions in comments . I suggested to him that we can put the URL of the readme file here.